supplier_stock/models/supplier_stock.py

```diff
@@ -84,6 +84,23 @@ class SupplierStock(models.Model):
         related='product_id.product_tmpl_id.image_small',
         readonly=True,
     )
+    partners_note = fields.Text(
+        string = 'Partner Note',
+    )
+    # Independent of current currency rate
+
+    domestic_retail_in_currency = fields.Float(
+        string='Retail in Currency',
+        required=True,
+        digits=dp.get_precision('Product Price'),
+    )
+    discount_in_curr = fields.Float(
+        string='Discount in currency',
```

Comment:
It sounds a bit strange to me as discount does not really have a currency, maybe `Discount of XXXX` is more understandable for user.

Reply:
The currency is specified by the field currency_id. But i added a unit of measurement.